src/components/web-monitoring-ui.tsx

```diff
@@ -57,3 +71,18 @@ export default class WebMonitoringUi extends React.Component<undefined, IWebMoni
         return {api};
     }
 }
+
+function getPagesByUser (userName: string): Promise<any> {
+    const url = `${window.location.origin}/domains/${userName}`;
```

Comment:
You can actually fetch on the current host with a URL beginning with `/`, so you should be able to leave out the `${window.location.origin}` part entirely.